akvo/rsr/usecases/remove_project_parent.py

```diff
@@ -4,13 +4,16 @@
 # See more details in the license.txt file located at the root folder of the Akvo RSR module.
 # For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
 
-from akvo.rsr.models import RelatedProject
+from akvo.rsr.models import Project
 from akvo.rsr.usecases.change_project_parent import RF_MODELS_CONFIG
 
 
-def remove_parent(project):
+def remove_parent(project: Project):
+    if not project.has_ancestors:
+        return
+
+    # Update related objects
     for (model, parent_attr, project_relation) in RF_MODELS_CONFIG.values():
         model.objects.filter(**{project_relation: project}).update(**{parent_attr: None})
 
-    RelatedProject.objects.filter(related_project=project, relation='2').delete()
-    RelatedProject.objects.filter(project=project, relation='1').delete()
+    project.delete_parent(force=True)
```

Comment:
`.save()`?

Reply:
Fixed in 086c9ca39ed0327d2c6b33a91903fd69b88318bf